src/components/v5/shared/AvatarUser/AvatarUser.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+
+import Avatar from '~v5/shared/Avatar';
+import Link from '~v5/shared/Link';
+import { AvatarUserProps } from './types';
+
+const displayName = 'v5.AvatarUser';
+
+const AvatarUser: FC<AvatarUserProps> = ({
+  walletAddress,
+  avatar,
+  isContributorsList,
+  userStatus,
+  userName,
+  size,
+  isLink = false,
```

Comment:
Instead of adding such prop, please just pass a href prop and based on this prop add a link or not. The URL shouldn't be hardcoded here